src/content/contributing/translation-program/index.md

```diff
@@ -20,6 +20,8 @@ The ethereum.org Translation Program is open and anyone can contribute!
 4. Machine translations will not be approved.
 5. All translations are reviewed before being added to the site, so there will be a short delay before your translations go live.
 
+_Join the [ethereum.org Discord](https://discord.gg/6WX7E97) to collaborate on translations, ask questions, share feedback and ideas, or join a translation group._
```

Comment:
@lukassim are you wanting to track the conversion here? if not, we should use `ethereum.org/discord`